### PR TITLE
[Attribute] Configuration validation of text and select attributes

### LINKED ISSUE #8671: Attribute of type text allows minLength greather than maxLength
Logged as an administrator;
Accessing admin panel;
Acessing attributes;
Clicking on create > Text;

I can fill Min length with a greater value than Max length and successfully save the attribute.
Later when I'll add a product with that attribute, the validation prevents me from save the product since the value is greater than max length to fulfill the min length requirement.

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | -----
| Branch?         | 1.0
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #8671
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Context/Ui/Admin/ManagingProductAttributesContext.php [hunk 1]
```diff
@@ -273,6 +273,40 @@ function () use ($productAttribute, $oldValue, $newValue) {
     }
 
     /**
+     * @When I specify its min length as :min
+     * @When I specify its min entries value as :min
+     */
+    public function iSpecifyItsMinValueAs(int $min)
```
Comment (Zales0123) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingProductAttributesContext.php]: `: void` R.I.P [*] 😢

File: src/Sylius/Bundle/AttributeBundle/Validator/Constraints/ValidSelectAttributeConfiguration.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AttributeBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+class ValidSelectAttributeConfiguration extends Constraint
```
Comment (pamil) [inline src/Sylius/Bundle/AttributeBundle/Validator/Constraints/ValidSelectAttributeConfiguration.php]: final?

File: src/Sylius/Bundle/AttributeBundle/Validator/Constraints/ValidTextAttributeConfiguration.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AttributeBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+class ValidTextAttributeConfiguration extends Constraint
```
Comment (pamil) [inline src/Sylius/Bundle/AttributeBundle/Validator/Constraints/ValidTextAttributeConfiguration.php]: final?

**Comment by pamil** [comment]
Since that specs have been made final, we should replace creating mocks of them with using these concrete classes in validator specifications.

**Comment by GSadee** [comment]
@pamil fixed 👍

**Comment by pamil** [comment]
Thanks @GSadee! 🎉

**Pull request merged**